Fix tag buttons; ensure is-success color isn't overriden

obulat opened the pull request:
## Fixes
<!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
<!-- If there is no issue being resolved, open one before creating this pull request. -->
Fixes #799 by @zackkrida 

## Description
<!-- Concisely describe what the pull request does. -->
This PR fixes tag, removable tag, and is-success button colors. 

## Screenshots
<!-- Add screenshots to show the problem and the solution; or delete the section entirely. -->
Normal state:
![image](https://user-images.githubusercontent.com/15233243/100220606-74a35780-2ecc-11eb-991b-5fedac4ada96.png)
![image](https://user-images.githubusercontent.com/15233243/100220578-69e8c280-2ecc-11eb-9b10-0ab253cf7c5d.png)
Hovered:
![image](https://user-images.githubusercontent.com/15233243/100220465-432a8c00-2ecc-11eb-8a9a-6fcb78df6650.png)
![image](https://user-images.githubusercontent.com/15233243/100220508-52113e80-2ecc-11eb-9377-4063012fafd3.png)


This PR also sets the background color to 'is-success' button to ensure that its color is not overridden by other libraries.
## Checklist
<!-- Replace  the [ ] with [x] to check the boxes. --> 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin
<!-- You must read and understand the following attestation. -->

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

neeraj-2 reviewed (commented):
thanks, @obulat, I was not able to work on it, BCS of some other work

obulat commented:
You are welcome, @neeraj-2 ! I needed the 'is-success' buttons to be the correct green, so I fixed that and added the tags fix since I'm working on it anyway. 

> I think it is still broken.🤔
> 
> ![image](https://user-images.githubusercontent.com/56186425/100232717-bc6ac300-2f4e-11eb-8f68-0f5776f57b5c.png)

What is your setup? I can't reproduce this, and I can see that I added background color in the 'files changed' tab here...🤔

zackkrida reviewed (approved):
Thank you!
